apps/i18n/common/en_us.json

```diff
@@ -175,9 +175,9 @@
   "assessmentSettings": "Assessment Settings",
   "assign": "Assign",
   "assignACourse": "Assign a course to your classroom or start your own course.",
-  "assignACurriculum": "Assign a curriculum",
   "assignCourse": "Assign Course",
   "assignConfirm": "Are you sure you want to assign \"{assignmentName}\" to \"{sectionName}\"?",
+  "assignCurriculum": "Assign Curriculum",
```

Comment:
that might be a benefit too, if the translations kept the article

Reply:
I checked in with Tess about this- I just assumed it would be cleaner to remove it because this is its only use, but I can see the argument for wanting to hold onto translations just in case!

Reply:
it's definitely cleaner to remove it :) I would just worry if we're launching to non-English users this week that we wouldn't have time to get this string translated. I'm not sure how big of a deal that is (that's up to Tess tbh), but with such a high profile feature I'm being a bit picky